enterprise/backend/test/metabase_enterprise/advanced_permissions/common_test.clj

```diff
@@ -12,6 +12,26 @@
             [metabase.test :as mt]
             [metabase.util :as u]))
 
+(defn- do-with-all-user-data-perms
```

Comment:
I just moved this from below, and made it a bit more robust (it now restores permission rows that were removed after a test)